src/Microsoft.OData.Core/JsonLight/ODataJsonLightPropertyAndValueDeserializer.cs

```diff
@@ -728,6 +703,67 @@ in resourceState.PropertyAndAnnotationCollector.GetODataPropertyAnnotations(nest
             return ODataJsonLightReaderNestedResourceInfo.CreateResourceSetReaderNestedResourceInfo(nestedResourceInfo, navigationProperty, propertyType, expandedResourceSet);
         }
 
+/*
+        /// <summary>
+        /// Reads a nested stream
+        /// </summary>
+        /// <param name="resourceState">The state of the reader for resource to read.</param>
+        /// <param name="property">The property for which to read the nested resource info. null for undeclared property.</param>
+        /// <param name="propertyName">The property name.</param>
+        /// <returns>The nested resource info for the stream collection.</returns>
+        /// <remarks>
+        /// This method doesn't move the reader.
+        /// </remarks>
+        protected static ODataJsonLightReaderNestedInfo ReadStreamNestedResourceInfo(IODataJsonLightReaderResourceState resourceState, IEdmStructuralProperty property, string propertyName)
+        {
+            Debug.Assert(resourceState != null, "resourceState != null");
+            Debug.Assert((propertyName != null) || (property != null),
+                "The collection property and property name shouldn't both be null");
+            Debug.Assert(property == null || property.Type.Definition.IsStream(),
+                "The type of the property, if specified, must be Edm.Stream");
+
+            ODataNestedResourceInfo nestedResourceInfo = new ODataNestedResourceInfo()
+            {
+                Name = propertyName,
+                IsCollection = false,
+                IsComplex = false
+            };
+
+            return ODataJsonLightReaderNestedResourceInfo.CreateResourceReaderNestedResourceInfo(nestedResourceInfo, property, EdmCoreModel.Instance.GetPrimitiveType(EdmPrimitiveTypeKind.Stream));
+        }
+*/
+
+        /// <summary>
+        /// Reads a nested stream collection as a resource set.

```

Comment:
>resource set [](start = 50, length = 12)

as a nested resource info?

Reply:
yes, as nested resource info representing a resource set. Clarified.

---
In reply to: [259081790](https://github.com/OData/odata.net/pull/1204#discussion_r259081790) [](ancestors = 259081790)